rtCommon/bidsIncremental.py

```diff
@@ -463,34 +463,27 @@ def removeMetadataField(self, field: str, strict: bool = False) -> None:
             self._exceptIfNotBids(field)
         self._imgMetadata.pop(field, None)
 
-    @property
-    def imageMetadata(self):
+    def getImageMetadata(self):
         return self._imgMetadata.copy()
 
-    @property
-    def suffix(self) -> str:
+    def getSuffix(self) -> str:
         return self._imgMetadata.get("suffix")
 
-    @property
-    def datatype(self) -> str:
+    def getDatatype(self) -> str:
```

Comment:
Should 'type' be camel case? getDataType()?

Reply:
It's presented as one word in the PyBids entities (e.g., `datatype='func'`), so I think treating it as one word is the most consistent.

I agree it is a bit odd though.